app/Http/Controllers/TaskResourceController.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\AddTaskRequest;
+use App\Task;
+use Session;
+
+class TaskResourceController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $task = Task::orderBy('created_at','asc')->get();
+
+        return view('tasks', [
+            'tasks' => $task,
+            'title' => trans('navbar.title') 
+        ]);
```

Comment:
Nghiên cứu ở đây để chọn ra cách passing data to view https://laravel.com/docs/5.5/views#passing-data-to-views